dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04205MigrateVanityURLToContent.java

```diff
@@ -59,7 +59,7 @@
     private static final int CODE_301 = 301;
     private static final int FOR_LIMIT_48 = 48;
     private static final int FOR_LIMIT_25 = 25;
-    private static final int INT_2 = 2;
+    private static final int INT_1 = 1;
 
     @Override
     public void executeUpgrade() throws DotDataException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This method has 76 lines, which is greater than the 75 lines authorized. Split it into smaller methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS138)